src/app/content/utils.spec.ts

```diff
@@ -198,9 +198,4 @@ describe('toRelativeUrl', () => {
     const url = toRelativeUrl(PAGE_URL, PAGE_URL + '/');
     expect(url).toMatchInlineSnapshot(`"page1"`);
   });
-
-  it('ignores trailing slashes on the source', () => {
-    const url = toRelativeUrl(PAGE_URL + '/', PAGE_URL);
-    expect(url).toMatchInlineSnapshot(`"page1"`);
-  });
```

Comment:
I don't think this is right. If the given url was `'book/a/page/b/'` and if `'b'` was returned then after clicking the relative url, the browser would navigate to `'book/a/page/b/b'`